packages/build/src/log/messages/plugins.js

```diff
@@ -36,15 +36,6 @@ const logFailPluginWarning = function (methodName, event) {
   )
 }
 
-const logPluginNodeVersionWarning = function ({ logs, pluginNames, userNodeVersion, currentNodeVersion }) {
-  logWarning(
-    logs,
-    `Warning: please ensure the following Build Plugins are compatible with Node.js ${currentNodeVersion}: ${pluginNames.join(
-      ', ',
-    )}. These plugins currently run on Node.js ${userNodeVersion}. Due to upcoming changes, they will soon run on the build system's Node.js version (currently ${currentNodeVersion}). For more info: http://ntl.fyi/build-plugins-node-versions`,
-  )
-}
-
```

Comment:
This warning is no longer applicable since we're now running all the affected plugins with our system Node version.